build_defs/python.build_defs

```diff
@@ -326,14 +326,25 @@ def python_test(name:str, srcs:list, data:list|dict=[], resources:list=[], deps:
         pre_cmd = worker_cmd,
     )
 
+    # N.B. the actual test sources are passed as data files as well. This is needed for pytest but
+    #      is faster for unittest as well (because we don't need to rebuild the pex if they change).
+    if isinstance(data, dict):
+        data['_srcs'] = srcs
+    else:
+        data.extend(srcs)
+
+    if interpreter.startswith('//') or interpreter.startswith(':'):
```

Comment:
`if looks_like_build_label(interpreter):` should do it

Reply:
Aha! I thought that there must be something like this but it's not documented